static/app/components/compactSelect/list.tsx

```diff
@@ -138,7 +145,7 @@ function List<Value extends SelectKey>({
   closeOnSelect,
   ...props
 }: SingleListProps<Value> | MultipleListProps<Value>) {
-  const {overlayState, registerListState, saveSelectedOptions, search} =
+  const {overlayState, registerListState, saveSelectedOptions, search, overlayIsOpen} =
```

Comment:
where does saveSelectedOptions come from now?

Reply:
`static/app/components/compactSelect/control.tsx` sets it.